package.json

```diff
@@ -55,7 +55,7 @@
   "scripts": {
     "start": "gulp fractal",
     "lint": "standard",
-    "test": "npm run lint && gulp test"
+    "test": "yarn run lint && gulp test"
```

Comment:
This means that regardless of whether they want to use npm or yarn, they at least have to have yarn installed to run the test suite right? Maybe we should get rid of the separate `lint` task for npm and change this to `standard && gulp test`, or use [gulp-standard](https://github.com/emgeee/gulp-standard)?